recipe/deploy/vendors.php

```diff
@@ -1,15 +1,33 @@
 <?php
 namespace Deployer;
 
+set('vendor_paths', ['{{release_or_current_path}}']);
 set('composer_action', 'install');
 
+// Self update composer from time to time
+// Set `false` to disable or an integer to update every N days
+// Default to 30 days
+set('composer_self_update', 30);
+
 set('composer_options', '--verbose --prefer-dist --no-progress --no-interaction --no-dev --optimize-autoloader');
 
-// Returns Composer binary path in found. Otherwise try to install latest
+// Returns Composer binary path if found. Otherwise try to install latest
 // composer version to `.dep/composer.phar`. To use specific composer version
 // download desired phar and place it at `.dep/composer.phar`.
 set('bin/composer', function () {
-    if (test('[ -f {{deploy_path}}/.dep/composer.phar ]')) {
+    $composer_phar_exists = test('[ -f {{deploy_path}}/.dep/composer.phar ]');
+    if ($composer_phar_exists) {
+        // If composer.phar is older than `composer_self_update` days, run self update
+        if(
+            get('composer_self_update')
```

Comment:
Using `0` as second parameter would make this operator superfluous

Reply:
done!